core/src/main/scala/com/salesforce/op/stages/impl/regression/OpRandomForestRegressor.scala

```diff
@@ -125,4 +127,9 @@ class OpRandomForestRegressionModel
   ttov: TypeTag[Prediction#Value]
 ) extends OpPredictionModel[RandomForestRegressionModel](
   sparkModel = sparkModel, uid = uid, operationName = operationName
-)
+){
+  @transient lazy protected val predict: Vector => Double = getSparkMlStage().map(s => s.predict(_))
```

Comment:
This also seems to be a very repetitive pattern. We can add a helper method for it as well.

Reply:
so the problem with a helper function in this one is there is no shared class for the mleap regressors that contains the predict function. they all implement it but you have to cast them to their specific type to get the predict. Thus a helper function only saves one map and makes it hard to read. I suppose I could use reflection in a shared helper...do you think that is better?